src/main/api/2/studio-api-2.yaml

```diff
@@ -4085,6 +4085,39 @@ paths:
 #        '500':
 #          $ref: '#/components/responses/InternalServerError'
 
+
+  /translation/config:
+    get:
+      tags:
+        - translation
```

Comment:
This is a translation configuration API, so it should be under configuration space, no ?

Reply:
@sumerjabri what do you think?

Reply:
After reviewing the `ConfigurationController` (namespace `/api/2/configuration`) it is a generic read/write controller, this one just reads a specific section of `site-config.xml` so I think they should be separated

Reply:
As our goal/target is to have all configuration in a single file at some point, small steps towards it are to keep services in a single namespace/controller

Reply:
Let's use a new API as `/api/2/configuration/{siteId}/{module}/{section}/{key}/{value}` with GET/POST.
Params:
module (in), key (in, optional), JSON (in/out).

You can read a section and write a whole section or a parameter in a section.

Example:

`GET /api/2/configuration/{siteId}/studio/translation` -> returns the translation section
`GET /api/2/configuration/{siteId}/studio/translation/defaultLocale` -> returns the default locale for this site
`POST /api/2/configuration/{siteId}/studio/translation/localeCodes` -> pass in the localeCodes JSON and it'll write it

For global configuration, it will be a reserved keyword, like `studio_root` or similar (since we already have that).

In the near-term, we will only support reading whole sections, and we can move to support writing and key access in the future. So only implement the GET.

This will change the UI behavior over time to think about config as params and not as files. This will help moves us towards our final goal of a new generalized configuration service independent of files.